src/main/java/uk/ac/ebi/eva/evaseqcol/dus/NCBIBrowser.java

```diff
@@ -131,4 +131,14 @@ public FTPFile getNCBIAssemblyReportFile(String directoryPath) throws IOExceptio
 
         return assemblyReport.orElseThrow(() -> new AssemblyNotFoundException("Assembly Report File not present in given directory: " + directoryPath));
     }
+
+    /**
+     * Return a pointer to the assembly sequences' fasta file that will be downloaded*/
+    public FTPFile getAssemblySequencesFastaFile(String directoryPath) throws IOException {
+        Stream<FTPFile> ftpFileStream = Arrays.stream(super.listFiles(directoryPath));
+        Stream<FTPFile> assemblyReportFilteredStream = ftpFileStream.filter(f -> f.getName().contains("genomic.fna.gz") && !f.getName().contains("from"));
```

Comment:
`Stream<FTPFile> assemblyReportFilteredStream = ftpFileStream.filter(f -> f.getName().toLowerCase().contains("genomic.fna.gz") && !f.getName().toLowerCase().contains("from"));`

Reply:
btw, I am wondering why the "from" filter is needed?

Reply:
@sundarvenkata-EBI @tcezard 
Some assemblies have **multiple** Fasta files in their home path that ends with _genomic_fna.gz.
Eg [here](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.40_GRCh38.p14/)

![ambiguous-fasta](https://github.com/EBIvariation/eva-seqcol/assets/82417779/34d46328-367c-4471-9b0b-f1091bc16adc)

So I assumed that the one we're concerned about is the one that ends only with _**genomic_fna.gz (no _from_..)**, which also has the largest file size, based on other assemblies that has only one Fasta file and its name ends with **_genomic_fna.gz**, like this [one](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.29_GRCh38.p3/).

It was a self observation, but I think I should have raised an issue about it just to make sure.

Reply:
I think we'll see that in GCF assembly much more than in GCA but it makes sense to keep the filter for now.
Can you just add a comment with a link to the example assembly for the next person wondering.

Reply:
@tcezard 
Okay I will.
In the method's doc comment right ?